tensorflow/core/ops/math_ops.cc

```diff
@@ -1688,6 +1688,45 @@ inputs: Must all be the same size and shape.
 
 #endif  // INTEL_MKL
```

Comment:
The registration of `RequantizePerChannel` and `RequantizationRangePerChannel` ops must be inside the `#ifdef INTEL_MKL` guard, because their kernel builders are only defined with config=mkl.

Reply:
The Op needs to be available outside of MKL also, as there is no TF(Eigen) Op available with the same name. This is required for pbtxt , else API tests will fail. let me know if this is not clear. Tomorrow in the meeting with Intel and Google AG Ramesh might add more or else I can explain in detail.

Reply:
these 2 tests will pass only the way I have it now for both Eigen and MKL
1.//tensorflow/core/kernels:mkl_requantize_ops_test
2.tensorflow/core/api_def/api_test

Reply:
Thank you for the explanation! I was under the impression that registering the op without having a kernel builder (in the non-MKL case) would have a problem. I agree that it works fine. Saw that a lot of other MKL code does it this way too. :)